src/game/engine/sidebar.cpp

```diff
@@ -370,11 +376,19 @@ SidebarClass::SidebarClass() :
     SidebarBit8(false),
     SidebarBit16(false)
 {
+    //todo, what is this setting and why is One_Time overwriting it?
```

Comment:
Position and dimensions of the sidebar for shape drawing functions which take the window type.